tensorflow/core/ops/array_ops.cc

```diff
@@ -679,9 +679,21 @@ reshape(t, [2, 4]) ==> [[1, 1, 2, 2]
 # tensor 't' has shape [3, 2, 3]
 # pass '[-1]' to flatten 't'
 reshape(t, [-1]) ==> [1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4, 5, 5, 5, 6, 6, 6]
-# -1 can also be used with higher dimensional shapes
+
+# -1 can also be used to infer the shape
+# -1 is inferred to be 9
```

Comment:
nit: Add a colon at the end of each line like this, so that it can be clearer whether it is talking about the example before or after it.